### ISSUE TITLE
Pithos does not expose dbus MPRIS interface

### ISSUE THREAD

**Issue by MarkLodato** [pithos/pithos#565]
Pithos 1.4.1 with flatpak (at least) is not exposing the MPRIS interface.

```bash
$ dbus-send --print-reply --dest=io.github.Pithos /org/mpris/MediaPlayer2 org.mpris.MediaPlayer2.Player.Play
Error org.freedesktop.DBus.Error.UnknownMethod: No such interface 'org.mpris.MediaPlayer2.Player' on object at path /org/mpris/MediaPlayer2
```

`d-feet` confirms that there is no `/org/mpris/MediaPlayer2 ` object path ([screenshot](http://i.imgur.com/KA0nNlI.png)).  Perhaps this is due to `flatpak-dbus-proxy`?

**Comment by TingPing** [comment]
Works as expected here. Is the plugin disabled in *Preferences > Plugins > Mpris*?

**Comment by MarkLodato** [comment]
Ah, didn't know I had to enable it! Can you update the [wiki](https://github.com/pithos/pithos/wiki/DBus-Interfaces) to explain this?

**Comment by TingPing** [comment]
It *should* be enabled by default.

**Comment by MarkLodato** [comment]
Yes, but if it gets turned off somehow, it would be nice to document that on the wiki.
